ipm_service/setup.py

```diff
@@ -0,0 +1,31 @@
+import glob
+import os
+
+from setuptools import find_packages
+from setuptools import setup
+
+package_name = 'ipm_service'
+
+setup(
+    name=package_name,
+    packages=find_packages(exclude=['test']),
+    data_files=[
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
+        ('share/' + package_name, ['package.xml']),
+        ('share/' + package_name + '/launch',
+            glob.glob('launch/*.launch')),
+    ],
+    install_requires=[
+        'launch',
+        'setuptools',
+    ],
+    zip_safe=True,
```

Comment:
```suggestion
    zip_safe=True,
    maintainer='Florian Vahl',
    maintainer_email='git@flova.de',
    description='A ROS2 Inverse Perspective Mapping Service',
```

Reply:
Do we want to add this here too? Isn't it a bit duplicated as normally the package.xml is used for this. If we would build using cmake these options wouldn't be there too.

Reply:
This information is usually included when you use ``ros2 pkg create``, and yes I agree that it's duplicated with the package.xml, but there's been some [discussion about deduplication](https://github.com/ros2/ros2/issues/1132).